client/lobby/SelectionTab.cpp

```diff
@@ -313,6 +322,15 @@ void SelectionTab::keyPressed(EShortcut key)
 
 void SelectionTab::clickDouble(const Point & cursorPosition)
 {
+	int position = getLine();
+	int itemIndex = position + slider->getValue();
+
+	if(itemIndex >= curItems.size())
+		return;
+
+	if(curItems[itemIndex]->isFolder)
+		return;
```

Comment:
Why return? Or we use single click for folders?

Also - getLine() can return -1 (judging from code below), so itemIndex might be -1 as well, potentially crashing on access to curItems[itemIndex]

Reply:
We use single click.

Other thing: fixed